clients/python/provider/superposition_provider/http_data_source.py

```
@@ -219,6 +219,13 @@ def supports_experiments(self) -> bool:
 
     async def close(self) -> None:
-        """Close the HTTP client."""
+        """Close the HTTP client and its underlying aiohttp session."""
         if self.client:
+            try:
+                http_client = getattr(self.client._config, "http_client", None)
+                session = getattr(http_client, "_session", None)
+                if session and not session.closed:
+                    await session.close()
+            except Exception:
+                pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)